CertificateTests/CertificateTest.cs

```diff
@@ -0,0 +1,169 @@
+using AventStack.ExtentReports.Reporter;
+using AventStack.ExtentReports;
+using NunitCompetition.Utilities;
+using NUnit.Framework;
+using NunitCompetition.Page;
+using OpenQA.Selenium;
+
+namespace NunitCompetition.CertificateTests
+{
+
+    [TestFixture]
+    public class Tests : CommonHook
+    {
+        public Tests()
+        {
+            Console.WriteLine("Tests class instance created.");
+        }
+        [OneTimeSetUp]
+        public void EducationTest()
+        {
+            extent.CreateTest("Tests for Certificate Feature");
```

Comment:
you should move this into a hooks class as this will be used for all test cases i saw you created this again in education test, the setup is fine to stay per test class level! I would say read up on the hooks for Nunit to understand at what point each of them trigger and this will help you understand why I am suggesting this change